go/ndt7-client/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/apex/log"
+	"github.com/apex/log/handlers/cli"
+	"github.com/m-lab/ndt7-clients/go/ndt7-client/client"
+)
+
+var hostname = flag.String("hostname", "localhost", "Host to connect to")
+var port = flag.String("port", "443", "Port to connect to")
+var insecure = flag.Bool("insecure", false, "Skip TLS verify")
+
+func main() {
+	log.SetHandler(cli.Default)
+	log.SetLevel(log.DebugLevel)
```

Comment:
it intuitively seems like these two lines belong in `init()` and not `main()` but I am not sure why.

Reply:
I think they don't. They configure the global `apex/log` logger. This strikes me as policy, not mechanism, and I argue that the policy depends on how you want to use this code. The current for `main` is optimised to be pleasant. Someone using the `client` package in another context will probably want to use the `json` logger. Also, the current verbosity is maximum, but others may want to have other levels of verbosity for other code.

Reply:
I retract my comment. :)